### PR TITLE
Add basic docs for tracing

### PR DESCRIPTION
**Pull request by kulmann**
More or less dumped from chat log, so this basically belongs to @refs

Adding basic documentation for how to get started with tracing.

### CODE REVIEW

File: docs/tracing.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+weight: 55
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs
+geekdocFilePath: tracing.md
+---
+
+We use [Jaeger](https://www.jaegertracing.io) for request tracing within oCIS. There are only a few steps required
+to get started:
+
+1. Start oCIS with tracing enabled: `REVA_TRACING_ENABLED=true bin/ocis server`. This will tell services to send
```
Comment (micbar) [inline docs/tracing.md:13]: Start `ocis-reva` in this case

Comment (kulmann) [reply docs/tracing.md:13]: :thinking: oCIS single binary starts all the reva processes and looking into jaeger ui, there are traces showing up from reva. Feels like an easier way than firing up all the ocis-reva processes individually.

Comment (kulmann) [reply docs/tracing.md:13]: Would be cool though, to add details on `OCIS_TRACING_ENABLED` as well and highlight the differences.

Comment (micbar) [reply docs/tracing.md:13]: correct. What I mean is: you start the single binary, but using this flag will enable tracing only for the reva subcommands.

Comment (micbar) [reply docs/tracing.md:13]: just wording.

Comment (kulmann) [reply docs/tracing.md:13]: To be honest, I thought that we only had reva tracing. Looked up the `OCIS_TRACING_ENABLED` env var after your comment. Unfortunately it doesn't have a default jaeger endpoint set. I will look up the endpoint and change the doc to using `OCIS_TRACING_ENABLED`. Why would one trace only reva services, when we have tracing for all services. ;-)

Comment (micbar) [reply docs/tracing.md:13]: proxy should have tracing too

Comment (micbar) [reply docs/tracing.md:13]: https://github.com/owncloud/ocis-proxy/blob/master/pkg/flagset/flagset.go#L61

Comment (micbar) [reply docs/tracing.md:13]: You could have the assumption, that `OCIS_TRACING_ENABLED` will switch on tracing for all services. But you need to enable them one by one.

Comment (kulmann) [reply docs/tracing.md:13]: that was indeed my assumption. Thanks for clarifying!

Comment (kulmann) [reply docs/tracing.md]: I changed the documentation to incorporate that every service has it's own set of flags for tracing configuration. Please check again.

**Comment by micbar** [comment]
please rebase. Should fix CI

**Pull request merged**